### PR TITLE
status: 2023q1: clusteradm: minor changes

### PR DESCRIPTION
**Pull request by grahamperrin**
Improve the appearance of each indented paragraph that falls below a list item (bullet point). 

Let's note that indented paragraphs are not a feature of AsciiDoc. (We have a good example of improving a report by not requiring indents, unfortunately I can not find the example.)

Add full stops. 

NIC.br uses uppercase 'NIC' at <https://nic.br/>.

Your.Org uses uppercase 'O' at <https://your.org/contact.php>.

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc [hunk 1]
```diff
@@ -12,22 +12,27 @@ In this quarter, the team has worked on the following:
...
++
 Besides the broken CI servers, we have a few old servers with broken disks and faulty PSUs.
 This task is in conjunction with the FreeBSD Foundation and donors/sponsors.
 * Deploy infrastructure to mirror the websites +
++
 Since the FreeBSD website is now mostly static, we have begun deploying infrastructure to mirror www.FreeBSD.org and docs.FreeBSD.org around the world in FreeBSD project-managed mirrors.
-* Create a new search database engine for internal FreeBSD.org searching needs like mailing list and docs
+* Create a new search database engine for internal FreeBSD.org searching needs like mailing list and docs.
 
 FreeBSD Official Mirrors Overview:
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:37]: With the colon here, should the subsequent paragraph become a list item? (Compare with uses of colons and list items above.)

Alternatively, use a subheading (without a colon).

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc [hunk 2]
```diff
@@ -12,22 +12,27 @@ In this quarter, the team has worked on the following:
 * Regular support for FreeBSD.org user accounts
 * Regular disk and parts support (and replacement) for all physical hosts and mirrors
 * Improve the PowerPC package builders +
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:14]: I feel that a full stop is needed in places such as this. 

The two sentences that follow do have full stops.

**Comment by dbaio** [comment]
Thanks for the thoughts and ideas; I'll submit an update soon.

**Comment by grahamperrin** [comment]
> … Is this the example you are serching for? [freebsd/freebsd-quarterly#563 (files)](https://github.com/freebsd/freebsd-quarterly/pull/563/files)

That was it, thanks.

**Pull request closed without merging**